src/components/layout/nav.js

```diff
@@ -0,0 +1,1362 @@
+// TODO: (discussion) make nav pure component, and move usage of nav to custom
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
```

Comment:
This file was removed from the upstream master branch, you can remove it as well.

Reply:
Thanks, remove it in upcoming update.